search-sentiments-map/public/scripts/trends-script.js

```diff
@@ -28,17 +28,23 @@ function setTopTrends() {
 
   for (let i = 0; i < trends.length; i++) {
     const trendElement = document.createElement('li');
-    trendElement.innerText = `${trends[i].trendTopic}`;
-    // Display the number of countries where the search topic is trending,
-    // when hovered.
-    trendElement.title = `Trending in ${trends[i].count} countries`;
-
+    trendElement.innerHTML = `${trends[i].trendTopic}`;
+    trendElement.id = 'trend' + i;
     // Show a certain number of trending topics by default and hide the rest.
     trendElement.className = i < NUM_SHOWN ? CLASSNAME_SHOWN : CLASSNAME_HIDDEN;
     trendElement.addEventListener('click', (event) => {
       setNewTrend(event.currentTarget.innerText);
     });
     trendsList.append(trendElement);
+
+    // Display the number of countries where the search topic is trending,
+    // when hovered.
+    let options = {
+      content: `Trending in ${trends[i].count} countries`,
+      placement: 'right',
+      trigger: 'hover',
+    }
+    $('#trend' + i).popover(options);
```

Comment:
Refactor this to be `'#' + trendElement.id`

Reply:
ok